src/h3/mod.rs

```diff
@@ -952,6 +1009,15 @@ impl Connection {
 
     /// Opens QPACK encoder and decoder streams, if not already opened.
     fn open_qpack_streams(&mut self, conn: &mut super::Connection) -> Result<()> {
+        self.open_qpack_encoder_stream(conn)?;
```

Comment:
I guess it's kind of unrelated, but couldn't we just have a single "open_uni_stream()" that takes the stream type and returns the stream ID, so then we could do e.g. `self.local_qpack_streams.encoder_stream_id = open_uni_stream(QPACK_ENCODER_STREAM_TYPE_ID);` or something like that?

Reply:
lol yes that is a much better suggestion